pkg/acsengine/addons.go

```diff
@@ -85,6 +85,11 @@ func kubernetesManifestSettingsInit(profile *api.Properties) []kubernetesFeature
 			"cloud-controller-manager.yaml",
 			profile.OrchestratorProfile.KubernetesConfig.UseCloudControllerManager != nil && *profile.OrchestratorProfile.KubernetesConfig.UseCloudControllerManager,
 		},
+		{
+			"kubernetesmaster-pod-security-policy.yaml",
+			"pod-security-policy.yaml",
+			profile.OrchestratorProfile.KubernetesConfig.EnablePodSecurityPolicy != nil && *profile.OrchestratorProfile.KubernetesConfig.EnablePodSecurityPolicy,
```

Comment:
`helpers.IsTrueBoolPointer` is better here (we'll do a follow-up PR to clean up other usage patterns before that convenience function was introduced, for example in the `kubernetesmaster-cloud-controller-manager.yaml` entry above

Reply:
updated.